model/src/main/kotlin/utils/SortedSetConverters.kt

```diff
@@ -28,6 +28,10 @@ import java.util.SortedSet
 import org.ossreviewtoolkit.model.Package
 import org.ossreviewtoolkit.model.Project
 
+class ComparableSortedSetConverter<T : Comparable<T>> : StdConverter<Set<T>, SortedSet<T>>() {
```

Comment:
IMO a goal of https://github.com/oss-review-toolkit/ort/issues/6235 also is to stop `data class`es to implement  `Comparable`. With than in mind, I believe we should have a dedicated converter here that is specialized for `CopyrightFinding` and, similar to `PackageSortedSetConverter` below, hard-codes the comparator to `compareBy<CopyrightFinding>({ it.statement }, { it.location })`.

Reply:
Our [comments overlapped](https://github.com/oss-review-toolkit/ort/pull/7000#discussion_r1194808739). If possible, I'd prefer to stop implementing `Comparable` already in this commit.

Reply:
Ok, let's continue discussion below then.